docs/advanced/data_spec.md

```diff
@@ -116,40 +94,6 @@ node_weight: float, Node weight.
 node_features: | type1:v1 v2;type2:v1 v2 |, Node feature vectors, type can be one of the following: {f: float, b: binary, i: integer}. There can be any number of values for each feature. There can only be a single vector for each feature type.
 neighbors: | int, int, float, int, features |, src_id, dst_id, edge_weight, edge_type and a feature vector in the same form as node_features.
 
-Graph `meta.txt` is as follows with all pieces of text replaced by integers,
```

Comment:
This is a different meta, it is part of binary, rather than input.

Reply:
Yes, users do not need to write a meta.txt file for tsv format.

Reply:
This is not a meta file for TSV format(which is meta.tsv), it is metafile for partitions(meta_{partition_id}.txt or meta.txt for entire graph).

Reply:
TSV uses meta.json. This data_spec doc is for user input formats not generated binary files, having meta.txt information here is confusing espessially where it was: https://github.com/microsoft/DeepGNN/blob/main/docs/advanced/data_spec.md#tsv-format.